ibis/pyspark/compiler.py

```diff
@@ -1368,6 +1399,37 @@ def compile_array_index(t, expr, scope, **kwargs):
     return F.element_at(src_column, index)
 
 
+@compiles(ops.ArrayConcat)
+def compile_array_concat(t, expr, scope, **kwargs):
+    op = expr.op()
+
+    left = t.translate(op.left, scope)
+    right = t.translate(op.right, scope)
+    return F.concat(left, right)
+
+
+@compiles(ops.ArrayRepeat)
+def compile_array_repeat(t, expr, scope, **kwargs):
```

Comment:
https://spark.apache.org/docs/latest/api/python/pyspark.sql.html#pyspark.sql.functions.array_repeat ?

Reply:
Array repeat will result in `[[1, 2], [1, 2], [1, 2]]` rather than `[1, 2, 1, 2, 1, 2]`.

Reply:
I think we can use https://spark.apache.org/docs/latest/api/python/pyspark.sql.html#pyspark.sql.functions.flatten

I think it will still be better than UDF

Reply:
I missed the flatten function. Agreed it would be better and updated.